transports/janus_websockets.c

```diff
@@ -1154,6 +1154,22 @@ static int janus_websockets_common_callback(
 			}
 			if(!g_atomic_int_get(&ws_client->destroyed) && !g_atomic_int_get(&stopping)) {
 				janus_mutex_lock(&ws_client->ts->mutex);
+                if (lws_send_pipe_choked(wsi)) {
```

Comment:
The whole section you added is indented using spaces instead of tabs: per the project code-style, please use tabs instead.